.travis.yml

```diff
@@ -5,10 +5,12 @@ go:
   - 1.8.x
   - 1.9.x
   - 1.10.x
+  - 1.11beta2
 
 install:
-  - go get -v ./...
+  - env GO111MODULE=on go get -v ./...
```

Comment:
While this is redundant in go 1.11, it is left here so that the build works in prior versions of go.

Reply:
This is still needed in go 1.11 so a good idea all around to put in.